### PR TITLE
Ports documentation - extended

### PR DESCRIPTION
**Pull request by LLdaniel**
As wished in #13 this PR is trying to provide an even more detailed explanation of a port change mentioning the recreation of the container and explicitly noting down the start parameter change.

Thank you btw. for your awesome repository which makes it easy to host an ASA server for me and my friends!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,7 +237,7 @@ Open the `docker-compose.yml` file again and edit the lines where the container
 ...
 ```
 
-Adjust the port to your liking, but make sure that you change both numbers (the one before and after the `:`). e.g. if you want to change the game port to `7755`, then this would be the result:
+Adjust the port to your liking, but make sure that you change both numbers (the one before and after the `:`). Assuming the above game port change to `7755` this would be the result:
```
Comment (mschnitzer) [inline README.md:240]: `change` -> `changes`

File: README.md [hunk 2]
```diff
@@ -210,8 +210,20 @@ As of today, ASA does no longer offer a way to query the server, so there's no q
 
 ## Changing the game port and RCON port
 
-You already learned that ports are defined by `ASA_START_PARAMS` in the `docker-compose.yml` file. This just tells the ASA server what ports to bind, but this alone is not enough and you need to apply
-the following changes as well.
+You already learned that ports are defined by `ASA_START_PARAMS` in the `docker-compose.yml` file. This just tells the ASA server what ports to bind.
+As a first step for port changes adjust the start parameters accordingly.
+
+E. g. if you want to change the game port from `7777` to `7755` your new start parameters would be like this:
```
Comment (mschnitzer) [inline README.md:216]: `would be like this` -> `would look like this`

File: README.md [hunk 3]
```diff
@@ -237,6 +249,9 @@ Adjust the port to your liking, but make sure that you change both numbers (the
 ...
 ```
 
+Now that your port changes are set you have to recreate your container. Therefore you need to use `docker-compose up -d` in order to apply your port changes.
```
Comment (mschnitzer) [inline README.md:252]: I think a comma is missing after `set`.

**Comment by mschnitzer** [comment]
Hi, thanks for providing this PR. I added some inline comments you might want to have a look at.

**Comment by LLdaniel** [comment]
Of course, thanks for you review. :+1:

File: README.md (hunk 3 above)
Comment (LLdaniel) [reply README.md]: addressed with [45e3cfc](https://github.com/mschnitzer/ark-survival-ascended-linux-container-image/pull/82/commits/45e3cfc808e52bdf1f1af888a395b8a238ddf458)

File: README.md (hunk 2 above)
Comment (LLdaniel) [reply README.md]: addressed with [45e3cfc](https://github.com/mschnitzer/ark-survival-ascended-linux-container-image/pull/82/commits/45e3cfc808e52bdf1f1af888a395b8a238ddf458)

File: README.md (hunk 1 above)
Comment (LLdaniel) [reply README.md]: addressed with [45e3cfc](https://github.com/mschnitzer/ark-survival-ascended-linux-container-image/pull/82/commits/45e3cfc808e52bdf1f1af888a395b8a238ddf458)

**Pull request merged**